tests/unit/components/settings/settings.spec.js

```diff
@@ -58,7 +58,7 @@ describe('NavBar', () => {
   test('Connected Edge device version is shown', () => {
     const localEdgeVersion = require('@/../package.json').version
 
-    store.state.version = localEdgeVersion
+    store.state.edgeDevice.version = localEdgeVersion
```

Comment:
@vickywane this comparison is meaningless. Compares edgeDeviceVersion to PWA version.